src/ORM/Connect/DBSchemaManager.php

```diff
@@ -800,13 +808,35 @@ public function requireField($table, $field, $spec)
                 }
             }
             $this->transAlterField($table, $field, $spec_orig);
+            if ($this->needRebuildColumn($fieldValue, $spec_orig)) {
+                if (!isset($this->schemaUpdateTransaction[$table]['advancedOptions'])) {
+                    $this->schemaUpdateTransaction[$table]['advancedOptions'] = [];
+                }
+                $this->schemaUpdateTransaction[$table]['advancedOptions'] = array_merge(
+                    $this->schemaUpdateTransaction[$table]['advancedOptions'],
+                    ['rebuildCols' => [$field => true]],
+                );
+            }
             $this->alterationMessage(
-                "Field $table.$field: changed to $specValue <i class=\"build-info-before\">(from {$fieldValue})</i>",
+                "Field $table.$field: changed to '$specValue' <i class=\"build-info-before\">(from {'$fieldValue'})</i>",
```

Comment:
Unrelated but it makes it easier to visually parse what's going on when indexes change
This change helped me debug